src/ngraph/op/util/rnn_cell_base.hpp

```diff
@@ -71,10 +73,48 @@ namespace ngraph
                 /// \return     The object representing activation function.
                 ///
                 ActivationFunction get_activation_function(std::size_t idx) const;
+                ///
+                /// \brief      Creates node with element-wise add operation with numpy broadcasting.
+                ///
+                /// \param[in]  lhs   The left hand side argument node.
+                /// \param[in]  rhs   The right hand side argument node.
+                ///
+                /// \return     Node with element-wise add operation.
+                ///
+                static std::shared_ptr<Node> add(const std::shared_ptr<Node>& lhs,
+                                                 const std::shared_ptr<Node>& rhs);
+                ///
+                /// \brief      Creates node with element-wise subtract operation with numpy broadcasting.
+                ///
+                /// \param[in]  lhs   The left hand side argument node.
+                /// \param[in]  rhs   The right hand side argument node.
+                ///
+                /// \return     Node with element-wise subtract operation.
+                ///
+                static std::shared_ptr<Node> sub(const std::shared_ptr<Node>& lhs,
+                                                 const std::shared_ptr<Node>& rhs);
+                ///
+                /// \brief      Creates node with element-wise multiply operation with numpy broadcasting.
+                ///
+                /// \param[in]  lhs   The left hand side argument node.
+                /// \param[in]  rhs   The right hand side argument node.
+                ///
+                /// \return     Node with element-wise multiply operation.
+                ///
+                static std::shared_ptr<Node> mul(const std::shared_ptr<Node>& lhs,
+                                                 const std::shared_ptr<Node>& rhs);
+                ///
+                /// \brief      Creates node with element-wise clip operation with numpy broadcasting.
+                ///
+                /// \param[in]  data   The input tensor for clipping.
+                ///
+                /// \return     Node with element-wise clip operation.
+                ///
+                std::shared_ptr<Node> clip(const std::shared_ptr<Node>& data) const;
 
             private:
-                std::size_t m_hidden_size = 0.f;
-                float m_clip = 0.f;
+                const std::size_t m_hidden_size = 0.f;
```

Comment:
Is this used anywhere?

Reply:
Yup, many times through this function [`get_hidden_size`](https://github.com/NervanaSystems/ngraph/pull/2983/files/0f8c8bf7430fd8ff2c09a74e993a8e307439c86b#diff-ca2772c2c3a02688d6215f75ff2da191R59). I.e [here](https://github.com/NervanaSystems/ngraph/pull/2983/files/0f8c8bf7430fd8ff2c09a74e993a8e307439c86b#diff-2ca43c9b3520dc9f4f25097d4351c299R219)